### PR TITLE
Add affinity photo cask

### PR DESCRIPTION
**Pull request by xeoneux**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by cho-m** [comment]
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Comment by xeoneux** [comment]
Thanks @cho-m for the help! Updated cask with zap 😄

File: Casks/affinity-photo.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask "affinity-photo" do
+  version "1.9.3"
+  sha256 :no_check
+
+  url "https://store.serif.com/download/075b51/"
+  name "Affinity Photo"
+  desc "Professional Image Editing Software"
```
Comment (cho-m) [inline Casks/affinity-photo.rb:7]: Minor style change to match other casks where (for `desc` stanza) we capitalize first word and proper nouns like a normal sentence (as opposed to `name` stanza which we capitalize like a title)
```suggestion
  desc "Professional image editing software"
```

Comment (xeoneux) [reply Casks/affinity-photo.rb]: Gotcha, will keep in mind 👍

**Review by cho-m — APPROVED** [review]
Thanks, @xeoneux

**Pull request merged**